plugin.json

```
@@ -128,6 +128,12 @@
             "type": "bool",
             "help_text": "When true, synthetic users will be converted to members when they login for the first time.",
             "default": false
+        },{
+            "key": "metricsForSharedChannelsInfrastructure",
+            "display_name": "Add metrics Shared Channels Service (Not used yet)",
+            "type": "bool",
+            "help_text": "When true, the system collect metrics about Shared Channels Service usage by the plugin without using it.",
+            "default": false
```

Can we default this to `true`? We ran into an issue recently where "new" config didn't get written to `config.json`, and then we couldn't manipulate it via `mmctl` until it had been set at least once. In this case, I think it's fine to default on and monitor as we rollout, and only turn it off if need be.

```suggestion
            "default": true
```